cmd/pcr0tool/commands/diff/command.go

```diff
@@ -5,8 +5,6 @@ import (
 	"encoding/json"
```

Comment:
Why do we have code changes in here

Reply:
As I said in Summary:

> Here we have two commits:
> * One to improve UX a little bit: to do not require `pcr0tool dump_registers` before `pcr0tool sum` on a CBnT machine.
> * Another is README.md

I was writing `README.md` and I realized that we overcomplicate things for an end user. So I decided to simplify a thing before documenting it.